example/intkey_multiply/processor/Cargo.toml

```diff
@@ -51,3 +51,6 @@ experimental = [
     "stable",
     # The following features are experimental:
 ]
+
+[patch.crates-io]
+transact = {git = "https://github.com/hyperledger/transact"}
```

Comment:
In most Cargo.toml files we put space after { and before }, so this would be:
```
transact = { git = "https://github.com/hyperledger/transact" }
```

Suggest we follow that convention here, despite the counter-example earlier in the file.

Reply:
fixed